### ISSUE TITLE
[WebUI] Markdown renderer doesn't work for inline code that is within `<details>   <summary>` tags

### ISSUE THREAD

**Issue by rmoff** [treeverse/lakeFS#5807]
Source code: 

    <details>
      <summary>CLI (local)</summary>
    We'll use the `lakectl` tool to create the branch. In a new terminal window run the following:
    
    ```bash
    lakectl branch create \
      lakefs://quickstart/denmark-lakes \
      --source lakefs://quickstart/main
    ```
    
    You should get a confirmation message like this:

Note that in the first line of text `lakectl` renders as \`lakectl\` and not `lakectl`

![image](https://user-images.githubusercontent.com/3671582/236267933-033fb802-4edc-4b84-aec4-2d75cc75ad05.png)

**Comment by rmoff** [comment]
@eladlachmi is this a quick fix?

**Comment by rmoff** [comment]
In fact, other markdown isn't rendering in these blocks either, such as emphasis: 

    _We'll use the `lakectl` tool to create the branch._ 
    
    In a new terminal window run the following:



![CleanShot_2023-05-04_at_21 55 34](https://user-images.githubusercontent.com/3671582/236327097-77b2aa2f-ee55-4394-9930-33b71564ac15.png)

**Comment by N-o-Z** [comment]
@rmoff Closing this as I believe it was resolved in #5976
Please re-open if you think necessary

**Comment by rmoff** [comment]
Confirmed; this is now working as it should 
![CleanShot_2023-06-21_at_14 39 25](https://github.com/treeverse/lakeFS/assets/3671582/cbebd6d6-0ccb-48c2-81b3-32f829ab4208)
